tests/e2e_regression/expected/db__runtime_txs.csv

```diff
@@ -327,7 +327,7 @@ emerald,8059761,7bf949dce8eb7b6bc6bafc33a60a1892300cebeb2668e5a2c369f61492659580
 emerald,8059762,4cd1b59d031c35e22f920ee13e7a6839a7d6515305d599a9b6c6e46c48717576,oasis1qqn5lhdlq7t730qyrsxj4ph4a8xnhwjtcs87fmrd,0,61294,consensus.Deposit,,,
 emerald,8059763,557e999ae7df04ea8a0974f1875a088ddc654ee36d694fbbd28ee838a82d0ae2,oasis1qzchk9eh0n4p2raym2e6cyvegrjumxl3lqkfhztr,2230600000000000,22136,evm.Call,,,
 emerald,8059765,babfe31abd744ede8e103bb2bfb66c8c860eead43552f814b1c759e6b0264ba8,oasis1qzchk9eh0n4p2raym2e6cyvegrjumxl3lqkfhztr,2230600000000000,22136,evm.Call,,,
-emerald,8059765,7cfa0143d9a5117bb8e94212ec30e183ea524625f1571c3cb87d4dfbdba679fc,oasis1qpgcp5jzlgk4hcenaj2x82rqk8rrve2keyuc8aaf,2928600000000000,29115,evm.Call,,,
+emerald,8059765,7cfa0143d9a5117bb8e94212ec30e183ea524625f1571c3cb87d4dfbdba679fc,oasis1qpgcp5jzlgk4hcenaj2x82rqk8rrve2keyuc8aaf,2928600000000000,29115,evm.Call,deposit,[],""
```

Comment:
This looks like a bug in parsing, but I haven't looked further. I did see a lot of warnings and errors in the logs from parsing. I'd like to submit as-is because the problem is unrelated to this PR, but we should follow up. @Andrew7234 can you have an initial look and create a ticket please? I think you'll be the most familiar with this part of the code.